pact.cabal

```diff
@@ -169,6 +169,7 @@ library
                      , safe >= 0.3.11 && < 0.4
                      , scientific >= 0.3.4.9 && < 0.4
                      , semigroups >= 0.18.2 && < 0.19
+                     , semigroupoids >= 5.0
```

Comment:
This is to get semigroup folds for anyone wondering

Reply:
Specifically `fold1`.